scripts/customize_build_env.sh

```diff
@@ -78,8 +81,8 @@ set_center_specific_modules()
             "ivybridge")
                 MODULE_CMD="module --force unload StdEnv; module load gcc/8.3.1 mvapich2/2.3 python/3.7.2"
                 ;;
-            "zen")
-                MODULE_CMD="module --force unload StdEnv; module load gcc/8.3.1 mvapich2/2.3 python/3.7.2"
+            "zen2")
+                MODULE_CMD="module --force unload StdEnv; module load gcc/8.3.1 mvapich2/2.3 python/3.7.2 opt rocm/3.10.0 hwloc/2.1"
```

Comment:
We will need hwloc@2.3+ because of this include: https://github.com/LLNL/Aluminum/blob/5dcf8756285caece49ffee97ce35d9d8d109a777/src/progress.cpp#L35